mrsal/mrsal.py

```diff
@@ -453,6 +437,8 @@ def start_consumer(self, queue: str, callback: Callable,
                         self.consumer_tag = method_frame.consumer_tag
                         app_id = properties.app_id
                         msg_id = properties.message_id
+                        if hasattr('headers', properties):
```

Comment:
Are you attempting to acknowledge a message irrespective of the result returned from the `callback` function? 
If that is the goal of the pull request, then this code is not accurate, and acknowledging the message from within the `consumer` itself is not considered good practice. Instead, the `callback` function should handle the acknowledgment and return true regardless of the processing result.